cypress/helpers/index.js

```diff
@@ -0,0 +1,5 @@
+export * from './constants'
+export * from './createUniqueDocument'
+export {default as sanityClientConfig} from './sanityClientSetUp'
+
+export const getTestId = (name) => `[data-testid="${name}"]`
```

Comment:
i like this helper and googled if there was something similar built in and found [custom commands](https://docs.cypress.io/api/cypress-api/custom-commands) where they have this example:

```js
Cypress.Commands.add('getBySel', (selector, ...args) => {
  return cy.get(`[data-test=${selector}]`, ...args)
})

// usage looks like `cy.getBySel('myTestId')`
```

how would you feel about making this a custom command? it might not work with intellisense though

Reply:
Great idea, yeah I was thinking about that! Would be better, seeing as we will use it across tests.